FieldComponent[] field_components
---
bool success
---
string message
